paywall/src/paywall-builder/constants.js

```diff
@@ -9,3 +9,9 @@ export const POST_MESSAGE_CONFIG = 'config'
 export const POST_MESSAGE_ACCOUNT = 'account'
 export const POST_MESSAGE_WEB3 = 'web3'
 export const POST_MESSAGE_WALLET_INFO = 'walletInfo'
+
+export const POST_MESSAGE_UPDATE_LOCKS = 'update/locks'
+export const POST_MESSAGE_UPDATE_ACCOUNT = 'update/account'
+export const POST_MESSAGE_UPDATE_ACCOUNT_BALANCE = 'update/balance'
```

Comment:
Nit For consistency I would like to see 'update/accountBalance' even though that probably does not matter ;)